install.sh

```diff
@@ -1,13 +1,12 @@
 #!/usr/bin/env bash
 set -eu
 
-VERSION="v0.3.0"
+VERSION="v0.4.0"
```

Comment:
This version doesn't exist but we need the path fix from #103, which will only be available in the next release.